apps/myaccount/src/public/deployment.config.json

```diff
@@ -32,16 +32,16 @@
         "enableOIDCSessionManagement": false
     },
     "session": {
-        "userIdleWarningTimeOut": 1740,
-        "sessionRefreshTimeOut": 28800,
-        "userIdleTimeOut": 1800,
-        "checkSessionInterval": 5
+        "userIdleWarningTimeOut": 580.0,
+        "sessionRefreshTimeOut": 300.0,
+        "userIdleTimeOut": 600.0,
```

Comment:
Hope these changes are intentional :)

Reply:
Yes. Syncing IS server actual values with the DEV server configs.